### ISSUE TITLE
AntiVirus threat detected

### ISSUE THREAD

**Issue by tobilen** [MgAl2O4/FFTriadBuddy#89]
<img width="561" alt="image" src="https://github.com/user-attachments/assets/a4e74b13-49ea-45db-9aea-e3d54cc0b14e">

anything to worry about?

**Comment by MgAl2O4** [comment]
Not really, sounds like a false alarm from antivirus heuristic detection. If you installed plugin via official Dalamud plugin manager, it's been reviewed and accepted as safe.

See antivirus section of Dalamud FAQ for details:
https://goatcorp.github.io/faq/xl_troubleshooting.html#q-how-do-i-whitelist-xivlauncher-and-dalamud-so-my-antivirus-leaves-them-alone

I'm very curios why it was flagged as trojan of all possible threats, it just reads game memory to understand state of TT mini game on screen. Isn't network comms a required component of any malware? Personally, I'd just whitelist entire Dalamud folder, as game injection itself may be seen as suspicious activity, but decision is up to you.

**Comment by MgAl2O4** [comment]
https://www.virustotal.com/gui/file/59c170bb05a5bd36b40089a03ecb7284950f35a6c9fc2b952254ac10e3f1a5c8/detection

I thrown plugin's dll file into virus total, version 1.10.0.1, same as on your screenshot. Looks like BitDefender & pretty much everyone else agree that it's safe stuff, but some vendors do think it's a malicious program :D
